docs/products/kafka/howto/kafka-sasl-auth.rst

```diff
@@ -1,18 +1,22 @@
 Use SASL Authentication with Apache Kafka®
-==========================================
+======================================================
 
-Aiven offers a choice of :doc:`authentication methods for Apache Kafka® <../concepts/auth-types>`, including `SASL <https://en.wikipedia.org/wiki/Simple_Authentication_and_Security_Layer>`_ (Simple Authentication and Security Layer).
+Aiven offers a selection of :doc:`authentication methods for Apache Kafka® <../concepts/auth-types>`, including `SASL <https://en.wikipedia.org/wiki/Simple_Authentication_and_Security_Layer>`_ (Simple Authentication and Security Layer).
 
-1. Scroll down the **Service overview** page to the **Advanced configuration** section and select **Change**.
-
-2. Turn on the setting labelled ``kafka_authentication_methods.sasl``, and click **Save advanced configuration**.
+1. Log in to `Aiven Console <https://console.aiven.io/>`_ and choose your project. 
+2. From the list of services, choose the Aiven for Apache Kafka service for which you wish to enable SASL.
+3. On the **Service overview** page of the selected service, scroll down to the **Advanced configuration** section. 
+4. Select **Change**.
+5. Enable the setting labeled ``kafka_authentication_methods.sasl``, and then select **Save advanced configuration**.
```

Comment:
`Enable the ``kafka_authentication_methods.sasl`` setting`

Reply:
Done.